foreign_cc/private/framework.bzl

```diff
@@ -646,6 +646,7 @@ def _copy_deps_and_tools(files):
         lines.append("##mkdirs## $$EXT_BUILD_DEPS$$/bin")
     for tool in files.tools_files:
         lines.append("##symlink_to_dir## $$EXT_BUILD_ROOT$$/{} $$EXT_BUILD_DEPS$$/bin/".format(tool))
+        lines.append("##path## $$EXT_BUILD_DEPS$$/bin/{}/bin".format(paths.basename(tool)))
```

Comment:
Can you explain the need for this?

Reply:
IIRC it was for autoconf as it makes unqualified calls out to tools such as autom4te - but it was so long ago I can't remember what issue I ran into that required this!

Reply:
Would you be willing to remove the line and see if you can reproduce the error?

Reply:
Yes; its running through CI now

Reply:
It passes without this line so leaving it out for now.